students/alexLaws/lesson07/test_html_render.py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env python3
+
+from html_render import Element
+from html_render import style_tags
+from html_render import Html
+from html_render import Head
+from html_render import Body
+from html_render import P
+from html_render import OneLineTag
+from html_render import Title
+from html_render import A
+from html_render import H
+from html_render import SelfClosingTag
+from html_render import Hr
+from html_render import Br
+from html_render import Meta
+from html_render import Ul
+from html_render import Li
```

Comment:
you can also do 
`import html_render`
or
`from html_render import Li, Ul, Meta ..`